widgetssdk/src/main/res/values/attrs.xml

```diff
@@ -166,7 +166,7 @@
     <!-- Bot message action button text color -->
     <attr name="gliaBotActionButtonSelectedTextColor" format="reference" />
 
-    <attr name="gliaWhiteLabel" format="reference" />
+    <attr name="whiteLabel" format="boolean" />
```

Comment:
many attrs have `glia` prefix, whitelabel doesn't, whats the pattern here?

Reply:
did it as in iOS tho with lower case 🤞